sycl/doc/extensions/supported/sycl_ext_oneapi_assert.asciidoc

```diff
@@ -138,8 +131,8 @@ enum class aspect {
 
 If device has the `ext_oneapi_native_assert` aspect, then its Device-Side
 Runtime is capable of native support of `assert`. That is, safe implementation
-is used. If device doesn't have the aspect, then fallback implementation is
```

Comment:
I think we could also remove the sentence:

> That is, safe implementation is used.

We don't define the term "safe implementation", so this sentence is not very clear.

Reply:
Good eye! I think it is the terminology used in the design document, which is no excuse. It has been removed.